src/Translation_unit.ml

```diff
@@ -83,6 +83,7 @@ let parse parse_ast (conf : Conf.t) ic =
   in
   Warnings.parse_options false (W.to_string warnings) ;
   let lexbuf = Lexing.from_channel ic in
+  Lexer.skip_hash_bang lexbuf ;
```

Comment:
We should be able to know how much was skipped looking at the lexbuf. That way, we would be sure we don't drop anything.

Reply:
It should be addressed by my last commit.